tests-hivemq-platform-operator/src/integrationTest/java/com/hivemq/helmcharts/compose/HelmUpgradeExtensionIT.java

```diff
@@ -181,11 +177,7 @@ void withBridgeConfiguration_updateExtensionWithNewConfig() throws Exception {
                 client.apps().statefulSets().inNamespace(namespace).withName(customResourceName).get();
         assertThat(upgradedStatefulSet.getStatus().getAvailableReplicas()).isEqualTo(1);
 
-        HELM_CHART_CONTAINER.uninstallRelease(PLATFORM_RELEASE_NAME,
-                "--cascade",
-                "foreground",
-                "--namespace",
-                namespace);
+        HELM_CHART_CONTAINER.uninstallRelease(PLATFORM_RELEASE_NAME, namespace);
```

Comment:
Do we need the try-finally contruct here as well?

Reply:
Refactored some ITs to remove boilerplate code, so it also deletes the namespace inside the `uninstallRelease` method.